routers/repo/projects.go

```diff
@@ -270,23 +270,19 @@ func ViewProject(ctx *context.Context) {
 		return
 	}
 
-	uncategorizedBoard, err := models.GetUncategorizedBoard(project.ID)
-	uncategorizedBoard.Title = ctx.Tr("repo.projects.type.uncategorized")
-	if err != nil {
-		ctx.ServerError("GetUncategorizedBoard", err)
-		return
-	}
-
 	boards, err := models.GetProjectBoards(project.ID)
 	if err != nil {
 		ctx.ServerError("GetProjectBoards", err)
 		return
 	}
 
-	allBoards := models.ProjectBoardList{uncategorizedBoard}
-	allBoards = append(allBoards, boards...)
+	for i := range boards {
+		if boards[i].ID == 0 {
+			boards[i].Title = ctx.Tr("repo.projects.type.uncategorized")
+		}
```

Comment:
add a break

Reply:
done

Reply:
Now realistically we know that the ID 0 board can only ever be the first board in the list so why can't we just check that and add an assertive comment in getBoards explaining that you must keep this the same?

Reply:
we can ... I'm not happy how translation is injected but I dont have a better idear for now

will add a comment and only check first